### ISSUE TITLE
Include conditional-flow rules

### ISSUE THREAD

**Issue by MaxTru** [camunda/linting#3]
**Problem**

Camunda 8 requires conditional flows (ie. sequence flows after conditional gateways) to either have a condition or be default.

This is not checked by camunda/linting yet, but would result in a deploy error

**Solution**

Deliver the existing https://github.com/bpmn-io/bpmnlint/blob/master/rules/conditional-flows.js rule per default. This rule will catch the problem case described above.

**Comment by barmac** [comment]
This is easy now since we ship `bpmnlint:correctness`. However, we need to still implement the properties panel integration.

**Comment by Buckwich** [comment]
I think this was already fixed by https://github.com/camunda/linting/pull/18, but its not using the exact same rule as `conditional flow`: 

<img width="1504" alt="Image" src="https://github.com/user-attachments/assets/dd2fddee-f5be-4da4-bc4c-39d58bfdc22b" />

**Comment by barmac** [comment]
Indeed, this was implemented via https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/97
